lingvo/core/ops/assert_kernels.cc

```diff
@@ -103,10 +103,10 @@ class AssertSameDim0Op : public OpKernel {
 REGISTER_KERNEL_BUILDER(Name("AssertSameDim0").Device(DEVICE_CPU),
                         AssertSameDim0Op);
 
-#if GOOGLE_CUDA
+// #if GOOGLE_CUDA
```

Comment:
If it is not necessary, can we revert this change? I dislike modifying core parts of the library, in case we might mess something up later.

Reply:
agreed